packages/core/src/queries/organization/index.ts

```diff
@@ -220,7 +220,7 @@ class OrganizationInvitationsQueries extends SchemaQueries<
         return sql`and ${fields.inviterId} = ${id}`;
       })}
       ${conditionalSql(invitee, (email) => {
-        return sql`and ${fields.invitee} = ${email}`;
+        return sql`and lower(${fields.invitee}) = lower(${email})`;
```

Comment:
question: is our `GET /users` API use lowercase comparison by default?

Reply:
You mean the user search API? The search API is case insensative by default